README.md

```diff
@@ -104,8 +104,8 @@ Main files:
 
 * **routes.js**: the file where views and routes are matched.
 * **views/Main.js**: the general component that defines the layout of the application.
-* **config/initialStateSlice.js**: the file that define the configuration of CARTO as default values for the slices. Set your CARTO account, apiKeys, basemap, OAuth apps, etc...
-* **config/appSlice.js**: general slice of the app to include/extend with custom app functionality.
+* **store/initialStateSlice.js**: the file that define the configuration of CARTO as default values for the slices. Set your CARTO account, apiKeys, basemap, OAuth apps, etc...
```

Comment:
If this is related why hygen? Could you remove this from this PR to include only hygen stuff?

Reply:
@alasarr hygen has a new slice generator, it needs a new store path, it's related. But I can divide this PR in two although the second one is needed to complete this.

Reply:
Done